js/runtime.go

```diff
@@ -124,21 +124,27 @@ func (r *Runtime) extractOptions(exports otto.Value, opts *lib.Options) error {
 }
 
 func (r *Runtime) loadFile(filename string) (otto.Value, error) {
```

Comment:
I'll try implementing this along with a paranoid test case, and see how it works